.python-version

```diff
@@ -1 +1 @@
-3.6.8
\ No newline at end of file
+3.6.9
```

Comment:
Really minor sorry, but can you add `.python-version` to the .gitignore file and delete this from the PR